packages/core/src/components.d.ts

```diff
@@ -51,24 +51,11 @@ export namespace Components {
         "type": 'submit' | 'reset' | 'button';
     }
     interface AtomCarousel {
-        "autoplay"?: boolean;
-        "autoplayDelay"?: AutoplayOptions['delay'];
-        "centerInsufficientSlides"?: boolean;
-        "centeredSlides"?: boolean;
-        "freeMode"?: boolean;
-        "loop"?: boolean;
-        "navigation"?: boolean;
-        "navigationButtonSize"?: 'medium' | 'xxlarge';
-        "pagination"?: boolean;
-        "paginationClickable"?: boolean;
-        "paginationType"?: PaginationOptions['type'] | 'thumbnails';
-        "slidesPerGroup"?: number | string;
-        "slidesPerView"?: number | string;
-        "spaceBetween"?: number;
-        "speed"?: number;
-        "thumbnailImages"?: string;
-        "videoIcons"?: boolean;
-        "watchSlidesProgress"?: boolean;
+        "autoplay": number;
```

Comment:
(dúvida) Por que precisa de um número? Não pode ser um booleano? Ou é um delay? Na documentação não ficou claro

Reply:
improve it as [5ec8c32](https://github.com/juntossomosmais/atomium/pull/434/commits/5ec8c321f4c6dca5576e62200cd856ab5d4f2de4)